css/css-backgrounds/background-clip/clip-text-multi-line.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
```

Comment:
I don't think you want reftest-wait here -- it also looks like nothing removes it.  Same for the reference.